de-lib/src/main/java/org/iplantc/de/client/services/impl/AppUserServiceFacadeImpl.java

```diff
@@ -175,10 +175,7 @@ public void getAppDetails(final App app, DECallback<App> callback) {
         deServiceFacade.getServiceData(wrapper, new DECallbackConverter<String, App>(callback) {
             @Override
             protected App convertFrom(String object) {
-                Splittable split = StringQuoter.split(object);
-                AutoBean<App> appAutoBean = AutoBeanUtils.getAutoBean(app);
-                AutoBeanCodex.decodeInto(split, appAutoBean);
-                return appAutoBean.as();
+                return AutoBeanCodex.decode(factory, App.class, object).as();
```

Comment:
I wonder if this change introduced the bug that the "Fix NPEs in AppDetailsDialog" commit fixed?
It's too bad that this technique won't work anymore. Unless there's another way of fixing this date parsing error? Maybe changing the AutoBean's field type from a Date (or changing the value returned by the service)?